packages/components/focus-trap/__tests__/focus-trap.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('<ElFocusTrap', () => {
     </div>`,
```

Comment:
tsx should be used here

Reply:
This one is not in the refactor scope, the code changed here is meant to pass the test, will be refactored later.